### PR TITLE
ci: add git-utimes from git-extras

### LINKED ISSUE #4: Noise in website builds
https://github.com/r-dbi/r-dbi.org/commit/3319c142beb8a037ac0d34df404e608e80f9fe03

@maelle: Can we use the "Git changed" date instead? An easy way to accomplish this is `git utimes` from git-extras: https://github.com/tj/git-extras/blob/main/Commands.md#git-utimes .

### PR DESCRIPTION
**Pull request by maelle**
Fix #4

### CODE REVIEW

File: .github/workflows/render-website.yaml [hunk 1]
```diff
@@ -27,8 +27,19 @@ jobs:
           git config --global user.name "$GITHUB_ACTOR"
           git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      - name: Install git-extras
+        run: |
+          git clone https://github.com/tj/git-extras.git
+          cd git-extras
+          git checkout $(git describe --tags $(git rev-list --tags --max-count=1))
+          sudo make install
```
Comment (krlmlr) [inline .github/workflows/render-website.yaml:35]: Slightly better on your own machine, doesn't make a difference here:

```suggestion
          make && sudo make install
```

File: .github/workflows/render-website.yaml [hunk 2]
```diff
@@ -27,8 +27,19 @@ jobs:
...
+
       - uses: actions/checkout@v3
 
+      - name: Avoid noise in file metadata
+        run: |
+          git-utimes --newer
```
Comment (krlmlr) [inline .github/workflows/render-website.yaml:41]: Why do we need `--newer` ? Just curious.

**Review by krlmlr — APPROVED** [review]

File: .github/workflows/render-website.yaml (hunk 2 above)
Comment (maelle) [reply .github/workflows/render-website.yaml:41]: > The --newer flag preserves the original modification time of files that were committed from the local repo, by only touching files that are newer than their last commit date.

Maybe we don't need it? But that option just seemed to make sense. :sweat_smile:

File: .github/workflows/render-website.yaml (hunk 1 above)
Comment (krlmlr) [reply .github/workflows/render-website.yaml:35]: Ouch.

I mean, *normally* we'd `make` and `sudo make install` ...

File: .github/workflows/render-website.yaml [hunk 3]
```diff
@@ -27,8 +27,19 @@ jobs:
           git config --global user.name "$GITHUB_ACTOR"
           git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      - name: Install git-extras
+        run: |
+          git clone https://github.com/tj/git-extras.git
+          cd git-extras
+          git checkout $(git describe --tags $(git rev-list --tags --max-count=1))
+          make && sudo make install
```
Comment (krlmlr) [inline .github/workflows/render-website.yaml]:
```suggestion
          sudo make install
```

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**